### ISSUE TITLE
How to train network from scratch?

### ISSUE THREAD

**Issue by shahinsaleh** [lindawangg/COVID-Net#60]
Hello,

I want to train you network from scratch and not from the pre-trained "Small" and "Large" models.

Could you please describe how to do that since I want to compare the effects of data augmentation vs dropout vs no augmentation & no dropout?

Thank you!

**Comment by haydengunraj** [comment]
You can use the metagraph files for the pre-trained models and comment out the line(s) where the pre-trained weights are loaded. For example, in train_tf.py you would comment out line 83: `saver.restore(sess, os.path.join(args.weightspath, args.ckptname))`.
